src/main/webapp/resources/js/pages/admin/components/AdminUsersPage.jsx

```diff
@@ -13,23 +13,25 @@ import { PagedTableProvider } from "../../../components/ant.design/PagedTable";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import { AddNewButton } from "../../../components/Buttons/AddNewButton";
 import { UsersTable } from "../../../components/UsersTable/UsersTable";
+import CreateNewUser from "./user/CreateNewUser";
 
 export default function AdminUsersPage() {
   // The following renders the Users component view
```

Comment:
Can you update this to standard JSDoc

Reply:
Please see 60881d5a458730a644c50c3819f587c04ab833d0.